src/EnergyPlus/ChilledCeilingPanelSimple.cc

```diff
@@ -1569,33 +1577,36 @@ namespace CoolingPanelSimple {
 		// Initialize arrays
 		QCoolingPanelSurf = 0.0;
 		QCoolingPanelToPerson = 0.0;
-
+		
 		for ( CoolingPanelNum = 1; CoolingPanelNum <= NumCoolingPanels; ++CoolingPanelNum ) {
 
-			ZoneNum = CoolingPanel( CoolingPanelNum ).ZonePtr;
+			auto & ThisCP( CoolingPanel( CoolingPanelNum ) );
+			
+			ZoneNum = ThisCP.ZonePtr;
 			if ( ZoneNum <= 0 ) continue;
-			QCoolingPanelToPerson( ZoneNum ) += CoolingPanelSource( CoolingPanelNum ) * CoolingPanel( CoolingPanelNum ).FracDistribPerson;
+			QCoolingPanelToPerson( ZoneNum ) += CoolingPanelSource( CoolingPanelNum ) * ThisCP.FracDistribPerson;
 
-			for ( RadSurfNum = 1; RadSurfNum <= CoolingPanel( CoolingPanelNum ).TotSurfToDistrib; ++RadSurfNum ) {
-				SurfNum = CoolingPanel( CoolingPanelNum ).SurfacePtr( RadSurfNum );
-				if ( Surface( SurfNum ).Area > SmallestArea ) {
-					ThisSurfIntensity = ( CoolingPanelSource( CoolingPanelNum ) * CoolingPanel( CoolingPanelNum ).FracDistribToSurf( RadSurfNum ) / Surface( SurfNum ).Area );
+			for ( RadSurfNum = 1; RadSurfNum <= ThisCP.TotSurfToDistrib; ++RadSurfNum ) {
+				SurfNum = ThisCP.SurfacePtr( RadSurfNum );
+				auto & ThisSurf( Surface( SurfNum ) );
```

Comment:
These are all very good steps, just keep in mind that in the Utopian future, we'll have lower case variable names, so this would be `thisSurf`.  The reduction in array lookups here makes the code much more readable.

Reply:
@Myoldmopar Ah, I will definitely keep this in mind in the future.  Still working in old EnergyPlus F90 mode there.  Good thing that is an easy fix and I will make sure I start all variables with lowercase in future work.  Thanks for getting to this review so quickly!